addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -0,0 +1,134 @@
+odoo.define('website_slides.quiz.question.form', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
+var core = require('web.core');
+
+var QWeb = core.qweb;
+var _t = core._t;
+
+/**
+ * This Widget is responsible of displaying the question inputs when adding a new question or when updating an
+ * existing one. When validating the question it makes an RPC call to the server and trigger an event for
+ * displaying the question by the Quiz widget.
+ */
+var QuestionFormWidget = publicWidget.Widget.extend({
+    template: 'slide.quiz.question.input',
+    xmlDependencies: ['/website_slides/static/src/xml/slide_quiz_create.xml'],
+    events: {
+        'click .o_wslides_js_quiz_validate_question': '_validateQuestion',
+        'click .o_wslides_js_quiz_cancel_question': '_cancelValidation',
+        'click .o_wslides_js_quiz_add_answer': '_addAnswerLine',
+        'click .o_wslides_js_quiz_remove_answer': '_removeAnswerLine',
+    },
+
+    init: function (parent, options) {
+        this.$oldElem = options.oldElement;
+        this.question = options.question || {};
+        this.update = options.update;
+        this.sequence = options.sequence;
+        this.slideId = options.slideId;
+        this._super.apply(this, arguments);
+    },
+
+    start: function () {
+        var self = this;
+        return this._super.apply(this, arguments).then(function () {
+            self.$('.o_wslides_quiz_question input').focus();
+        });
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    _addAnswerLine: function (ev) {
+        $(ev.currentTarget).closest('.o_wslides_js_quiz_answer').after(QWeb.render('slide.quiz.answer.line'));
+    },
+
+    _removeAnswerLine: function (ev) {
+        if (this.$('.o_wslides_js_quiz_answer').length > 1) {
+            $(ev.currentTarget).closest('.o_wslides_js_quiz_answer').remove();
+        }
+    },
+
+    _validateQuestion: function (ev) {
+        var createNext = $(ev.currentTarget).hasClass('o_wslides_js_quiz_create_next');
+        this._createOrUpdateQuestion(createNext);
+    },
+
+    _cancelValidation: function () {
+        this.trigger_up('reset_display', {
+            $oldElem: this.$oldElem,
+            $elem: this.$el,
+            update: this.update
+        });
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    _createOrUpdateQuestion: function (createNext) {
+        var self = this;
+        var form = this.$('form');
```

Comment:
We usually prefix variables containing a jQuery element with $
So `$form` instead of `form` in this case.